### DISCUSSION TITLE
Cannot get Nx + Storybook + MSW to work together

### DISCUSSION THREAD

**Discussion by nareshbhatia** [mswjs/msw#973] [Q&A]
I have an app built using Nx. MSW is installed and working perfectly with the app. Also Jest tests are setup to work correctly with MSW. However Storybook refuses to work. It seems to find the handlers but has trouble loading mockServiceWorker.js. Here's the error message:

![image](https://user-images.githubusercontent.com/1220198/140651272-bcaf6861-a7dc-4bd1-92f4-2c7c0ae437ca.png)

Any idea what could be going wrong? Here's [my repo](https://github.com/nareshbhatia/react-test-shop-nx). `yarn storybook:app` to start storybook, then try CartView or CatalogView. You will see "Network Error".

Thanks in advance.

**Comment by voronindenis** [comment]
Need to add ```staticDirs: ['../public']``` to `.storybook/.main.ts`
